cmd/commands/server/command_server_test.go

```diff
@@ -0,0 +1,38 @@
+package server
+
+import (
+	"github.com/stretchr/testify/suite"
+	"github.com/mysterium/node/server"
+	"github.com/mysterium/node/identity"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"github.com/stretchr/testify/assert"
+)
+
+type testContext struct {
+	suite.Suite
+}
+
+var (
+	myID                  = identity.FromAddress("identity-1")
+	activeProviderID      = identity.FromAddress("vpn-node-1")
+	activeProviderContact = dto_discovery.Contact{}
+	activeProposal        = dto_discovery.ServiceProposal{
+		ProviderID:       activeProviderID.Address,
+		ProviderContacts: []dto_discovery.Contact{activeProviderContact},
+	}
+)
+
+func (tc *testContext) TestProposalUnregisteredWhenPingerClosed() {
```

Comment:
Simple TestXXX(t * testing.T) is enough here.
testContext structure is needed when you want to use Setup/Teardown callbacks before each test or test suite.